services/order.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+from django.db import transaction
+
+
+from db.models import Ticket, Order
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None
+                 ) -> QuerySet:
+    user = get_user_model().objects.get(username=username)

```

Comment:
Code Style: You are using single quotes in `get(username=username)`. It's better to use double quotes in the whole project for consistency. Replace it with `get(username=username)`.